### ISSUE TITLE
Add methods on the Users resource to update settings

### ISSUE THREAD

**Issue by Jericho** [Jericho/ZoomNet#190]
- Update user settings
- Update recording authentication
- Update meeting authentication
- Update security settings

**Comment by MaiklSemenov** [comment]
Hello @Jericho, thank you for the API library. Do you know when you are going to implement this?

**Comment by Jericho** [comment]
I put this feature on the back burner 2 years ago because I was overwhelmed by the number of settings that could be updated. I don't remember exactly how many settings were 'updatable' at the time I raised this setting but I checked the documentation this morning and the situation is even worse. There are now 8 categories (instead of just 4 as listed in my original comment):

- email notification: 7 settings
- feature: 28 settings
- meeting: 108 settings
- profile: 2 settings
- recording: 35 settings
- schedule meeting: 24 settings
- telephony: 4 settings
- tsp: 2 settings

That's a grand total of 210 different settings. It would not be particularly difficult to add this feature but it would be tedious, time consuming and error prone.

TL;DR : I do not know when this is going to be implemented but contributions are welcome!
